### PR TITLE
README refactor

### PR DESCRIPTION
**Pull request by jgollenz**
I'll open this so we can discuss how we could improve the current state of the README. Things I'd like to push for are: 

- Conciseness
- Showing > Describing
- Consistency

Changes I made so far ([have a look](https://github.com/jgollenz/orgmode/tree/refactor/readme)):

- Improved consistency for the code snippets
- Elevated some headlines so the structure closer resembles the top-level links
- Added 'Requirements' section
- Put 'Requirements', 'Installation', 'Setup' and 'Usage' under a 'Quickstart' section
- Renamed 'Gifs' to 'Showcase'
- Made the logo smaller
- Put the package managers and completion plugins into collapsible sections

Changes I propose:

- [ ] Combine the 'Features' and 'Showcase/Gifs' section. Have some strong gifs that show the main features of the port. A (semi-)full list of features can only grow in the future and blow up the README further. I think it's better to link to the docs. And IF we stay with such a list (exhaustive or not) in the README, we should link to the dedicated part of the documentation.
- [ ] I am a huge fan of plugins that just give me a display of their features first thing in the README. I only care about installation and setup when I think that the plugin does what I want.
- [x] Mention the Matrix channel
- [x] There is an inconsistency about the actual name of the repo. On one hand we have the organization `nvim-orgmode`, on the other the title of the repo in the README: `orgmode.nvim`. The repo itself is named `orgmode`. We should clarify this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,115 +119,119 @@ require('orgmode').setup({
...
     { mode = 'omni'},
   },
 }
+-- add additional keyword chars
 vim.cmd[[autocmd FileType org setlocal iskeyword+=:,#,+]]
 ```
 
+</details>
+
 Or just use `omnifunc` via <kbd>\<C-x\>\<C-o\></kbd>
```
Comment (jgollenz) [inline README.md:177]: @kristijanhusak I don't know if this line belongs to `completion-nvim`

File: README.md [hunk 2]
```diff
@@ -52,34 +90,25 @@ use {'nvim-orgmode/orgmode',
...
+</details>
 
-## Setup
+### Setup
 
 ```lua
 -- init.lua
 
--- Load custom tree-sitter grammar for org filetype
+-- Load custom treesitter grammar for org filetype
```
Comment (jgollenz) [inline README.md:100]: I chose to follow the wording of nvim-treesitter and not that from the original project called 'Tree-sitter'

**Review by jgollenz — COMMENTED** [review]
@kristijanhusak please let me know what you think of the changes I made so far.

I'm not yet satisfied with the sections 'Features', 'Usage' and 'Showcase'. As mentioned in the PR, I think 'Features' and 'Showcase' should be merged with just some good GIFs and a link to the docs. 'Usage' depends on the 'Getting started' page in the wiki

edit: perhaps it would be good to not wait until I get the time to make some GIFs I'm satisfied with. The sooner people see the link to the matrix chat the better

**Comment by kristijanhusak** [comment]
> perhaps it would be good to not wait until I get the time to make some GIFs I'm satisfied with. The sooner people see the link to the matrix chat the better

Looks good so far. I'm just curious why you added asset logo?

**Comment by jgollenz** [comment]
> I'm just curious why you added asset logo?

Because the other one was so huge and I am trying to make the README as short as reasonably possible. Also, that link was some magic number, now as part of the repo it looks a lot cleaner. But if you prefer it not to be part of the repo to keep the size small I can remove it ofc ;)

**Comment by kristijanhusak** [comment]
> Also, that link was some magic number, now as part of the repo it looks a lot cleaner

You can get a "free" storage of an image by uploading it to any github comment. Lets leave it as it is since there is no need to have an asset part of the repo.

File: README.md (hunk 1 above)
Comment (kristijanhusak) [reply README.md:177]: No, it should be separate paragraph. You can probably also put it in `details` tag.

**Comment by jgollenz** [comment]
@kristijanhusak btw, what is the actual name of the project? We currently have `nvim-orgmode` (organization), `orgmode.nvim` (headline in the README) and just `orgmode` (repo name) floating around.

**Comment by kristijanhusak** [comment]
Let's go with `nvim-orgmode`, since that is the organization name. `orgmode.nvim` is from the time I had the repo on my own personal profile.

**Comment by jgollenz** [comment]
I added a 'Getting Started' section (which perhaps should be called 'First Steps'? :thinking:) and linked it in the README. Let's just go ahead and merge this and deal with feature-list, roadmap and gifs/videos another day

**Comment by kristijanhusak** [comment]
Thanks!

**Pull request merged**